File: Modul 5 Sem 1/C/PRAK505-2410817110005-MuhammadRizqanKarim.c
#include <stdio.h>

void Biodata(int tahunLahir, char nama[], char asal[]) {
    int tahun_sekarang = 2020;
    int umur = tahun_sekarang - tahunLahir;
    printf("Perkenalkan Nama Saya : %s\n", nama);
    printf("Umur Saya : %d\n", umur);
    printf("Saya Adalah Angkatan : %d\n", tahun_sekarang);
    printf("Asal Saya dari : %s\n", asal);
}

int main() {
    int tahunLahir;
    char nama[20], asal[15];
    scanf("%d", &tahunLahir);
    scanf(" %[^\n]", nama);
    scanf(" %[^\n]", asal);
    Biodata(tahunLahir, nama, asal);
    return 0;
}

File: Modul 5 Sem 1/Phython/PRAK505-2410817110005-MuhammadRizqanKarim.py
def Biodata(tahunLahir, nama, asal):
    tahun_sekarang = 2020
    umur = tahun_sekarang - tahunLahir
    print(f"Perkenalkan Nama Saya : {nama}")
    print(f"Umur Saya : {umur}")
    print(f"Saya Adalah Angkatan : {tahun_sekarang}")
    print(f"Asal Saya dari : {asal}")

tahunLahir = int(input())
nama = input()
asal = input()
Biodata(tahunLahir, nama, asal)
